build/templates/session.py.mako

```diff
@@ -297,6 +315,15 @@ class Session(_SessionBase):
 %       endfor
         self._param_list = ', '.join(param_list)
 
+        # self.get_channel_names() and self.channel_count can only be called after the session
```

Comment:
suggestion - 

Store the list of channels in the Session which is needed by some `nimi-python` modules.
Use try/except because not all the modules support channels.

Reply:
I've added your suggested comment on top of my existing comment in my latest commit. Since they are explaining about different things, I decided to keep them both (mine is explaining about the need to put the try/except code after `self._vi` is populated).